apps/src/templates/Notification.jsx

```diff
@@ -107,13 +114,20 @@ const Notification = React.createClass({
     this.setState({open: !this.state.open});
   },
 
+  onAnnouncementClick: function () {
+    if (this.props.analyticId) {
+      trackEvent('teacher_announcement','click', this.props.analyticId);
```

Comment:
Just passing in a function and having all three parameters remain here in `Notification.jsx`?  Or passing in a new function which completely takes care of calling `trackEvent` with the appropriate parameters?